build.zig

```diff
@@ -495,16 +497,32 @@ fn build_ci_script(
     const run_artifact = b.addRunArtifact(scripts);
     run_artifact.addArgs(argv);
     run_artifact.setEnvironmentVariable("ZIG_EXE", b.graph.zig_exe);
-    build_ci_check_status(run_artifact);
+    hide_stderr(run_artifact);
     step_ci.dependOn(&run_artifact.step);
 }
 
-fn build_ci_check_status(step: *std.Build.Step.Run) void {
-    // NB: The purpose here is not to check status (Zig does that automatically), but rather to
-    // prevent inheriting stdio, which causes everything to execute sequentially due to stdio lock!
-    step.stdio = .{ .check = .{} };
```

Comment:
Without this, would the run steps now be sequential?

Reply:
No, they are still run in parallel. `.addCheck` already sets stdio to `.check`, so the explicit `step.stdio = .{ .check = .{} };` is simply unnecessary.